app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.getenv("API_KEY")
+FILTERING = "paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    weather_response = requests.get(URL + f"?key={API_KEY}&q={FILTERING}")
```

Comment:
Use `params` keyword argument for passing parameters instead of `f"?key={API_KEY}&q={FILTERING}"`.

Reply:
wdym?

Reply:
Method `get` has `params` keyword argument. Use its instead of `f"?key={API_KEY}&q={FILTERING}"`. You can find more details about it in the official documentation. [https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls](https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls)

Reply:
PARAMS = {"key": API_KEY, "q": FILTERING}
weather_response = requests.get(URL, params=PARAMS)
?